backend/Dockerfile

```diff
@@ -16,10 +16,14 @@ RUN apk add --no-cache \
 # switch to default user to not chown node_modules on startup
 USER $USER
 WORKDIR /app
-
 COPY --chown=$UID:$GID [ "package.json", "package-lock.json", "/app/" ]
+
+# run npm install as root, required for Windows hosts
+USER root
 RUN npm install
 
+RUN chown -R $UID:$GID ./
```

Comment:
Also, I'm chowning here at build-time to avoid it happening at run-time

Reply:
You should combine this line with the run where we do `npm install`. Otherwise if I remember correctly use double the size of `node_modules`.